modules/run-vault/run-vault

```diff
@@ -144,7 +166,24 @@ EOF
     consul_storage_type="ha_storage"
   fi
 
-  local readonly consul_storage=$(cat <<EOF
+
+  if [[ "$enable_dynamo" == "true" ]]; then
+    consul_storage=""
+    vault_storage=$(cat <<EOF
+storage "dynamodb" {
```

Comment:
In order to have both dynamodb and s3 backend we would need to be able to make that block similar to how its handled for consul.  Currently what I have adjusted on my end, and is working so far with manual testing:

```shell
if [[ "$enable_s3_backend" == "true" ]]; then
    s3_config=$(cat <<EOF
storage "s3" {
  bucket = "$s3_bucket"
  region = "$s3_bucket_region"
}\n
EOF
)
    consul_storage_type="ha_storage"
    dynamodb_storage_type="ha_storage"
  fi


  if [[ "$enable_dynamo_backend" == "true" ]]; then
    vault_storage=$(cat <<EOF
$dynamodb_storage_type "dynamodb" {
  ha_enabled = "true"
  region = "$dynamo_region"
  table  = "$dynamo_table"
}

# HA settings
cluster_addr  = "https://$instance_ip_address:$cluster_port"
api_addr      = "$api_addr"

EOF
)
```